hdfs/util.py

```diff
@@ -25,8 +25,9 @@ class HdfsError(Exception):
 
   """
 
-  def __init__(self, message, *args):
+  def __init__(self, message, *args, **kwargs):
```

Comment:
Let's just use `exception=None` instead of `**kwargs` since the constructor only uses this value.